File: Assignment1/program3.c
// 3. Program to print 5 to 1 on screen

#include <stdio.h>

void Display()
{
    int i = 0;
    i = 5;
    while(i>=1)
    {
        printf("%d",i);
        i--;
    }
}

int main()
{
    Display();

    return 0;
}

File: Assignment21/Program3.java
/* Write a program to display all factors of a given number*/
package Assignment21;

class Logic
{
    void displayFactors(int num)
    {
        int iCnt = 0;
        for(iCnt = 1 ; iCnt<=num/2; iCnt++)
        {
            if(num%iCnt == 0)
            {
                System.out.println(iCnt);
            }
        }
    }
}

class Program3 
{
    public static void main(String[] args) 
    {
        Logic obj = new Logic();
        obj.displayFactors(12);    
    }   
}
